src/TextBox/TextBox.tsx

```diff
@@ -47,15 +61,15 @@ export const TextBox: React.FunctionComponent<TextBoxProps> = (props: TextBoxPro
                     onKeyPress={props.onKeyPress}
                     onFocus={props.onFocus}
                     onBlur={props.onBlur}
-                    placeholder={props.placeHolder}
+                    placeholder={props.placeholder}
                     className="form-control"
                     autoFocus={props.focus}
-                    autoComplete={props.autoComplete ? "on" : "off"}
-                    readOnly={props.readonly}
+                    autoComplete={props.autoComplete}
+                    readOnly={props.readOnly}
                     disabled={props.disabled}
                     ref={props.reference}
                 />
-                <div className="alert alert-danger">{props.error}</div>
+                {showErrorMessage && !props.success ? <div className="alert alert-danger">{props.error}</div> : null}
```

Comment:
I would include the `!props.success` in the useEffect logic for showErrorMessage 

```
    React.useEffect(() => {
        setShowErrorMessage(!!props.showErrorMessage && !props.success);
    }, [props.showErrorMessage, props.success]);
```

Reply:
fixed in e90bd8f73c3e2216a90ccb30713b1707f1241063